client/blocks/signup-form/index.jsx

```diff
@@ -1061,14 +1065,19 @@ function TrackRender( { children, eventName } ) {
 }
 
 export default connect(
-	( state ) => ( {
+	( state, ownProps ) => ( {
 		oauth2Client: getCurrentOAuth2Client( state ),
 		sectionName: getSectionName( state ),
 		isJetpackWooCommerceFlow:
 			'woocommerce-onboarding' === get( getCurrentQueryArguments( state ), 'from' ),
 		isJetpackWooDnaFlow: wooDnaConfig( getCurrentQueryArguments( state ) ).isWooDnaFlow(),
 		from: get( getCurrentQueryArguments( state ), 'from' ),
 		wccomFrom: get( getCurrentQueryArguments( state ), 'wccom-from' ),
+		// displayUsernameInput:
+		// 	[ 'onboarding', 'personal', 'premium', 'business', 'ecommerce' ].includes(
+		// 		ownProps.flowName
+		// 	) && 'control' === abtest( 'removeUsernameInSignup' ),
```

Comment:
I am guessing this code is in progress?

Reply:
Yes! It was a temporary testing commit:

<img width="1042" alt="Screenshot 2020-09-15 at 7 12 16 PM" src="https://user-images.githubusercontent.com/1269602/93218077-71cb0e80-f787-11ea-8234-aefd2c42630b.png">